docs/drafts/Tasks/upgrading-or-downgrading-an-extension.md

```diff
@@ -0,0 +1,122 @@
+# Upgrading or Downgrading an Extension
+
+Existing extensions can be upgraded or downgraded by updating the version field in the ClusterExtension resource.
+
+## Prerequisites
+
+* You have an extension installed
+* The target version is compatible with OLM v1 (see [OLM v1 limitations](../refs/olmv1-limitations.md))
+* CRD compatibility between the versions being upgraded or downgraded (see [CRD upgrade safety](../../refs/crd-upgrade-safety.md))
+* The installer service account's RBAC permissions are adequate for the target version (see [Minimal RBAC for Installer Service Account](create-installer-service-account.md))
+
+For more detailed information see [Upgrade Support](../upgrade-support.md).
+
+## Procedure
+
+* Update the version field in the ClusterExtension resource:
+
+    ``` terminal
+    kubectl patch clusterextension <extension_name> --type='merge' -p '{"spec": {"source": {"catalog": {"version": "<target_version>"}}}}'
+    ```
```

Comment:
Something to consider: As a less experienced Kubernetes user, I find it easier to see the changes required in the CR. It does add an additional step of needing to run `kubectl apply -f` again, but it has the benefit of being able to see all the parameters and figuring out which ones I need to change.

I don't think this is a very big deal, and if it gets in the way of iterating quickly, please ignore it. Perhaps this is something we could do for our finalized documentation but not worry about at  this stage. :)

Reply:
![whynotboth](https://github.com/user-attachments/assets/96164db5-b2a4-4b5a-a7be-c0e01c38b81f)

Reply:
![whynotboth](https://github.com/user-attachments/assets/34e20105-d724-4806-9d98-bfea09cef92d)

Reply:
Downstream, I usually explain things the long way then add a TIP admonition for the faster, more efficient way. 

Here's a link to the markup in the MK Docs Material reference docs: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types